pkg/rpc/client/nep11.go

```diff
@@ -84,7 +84,7 @@ func (c *Client) CreateNEP11TransferTx(acc *wallet.Account, tokenHash util.Uint1
 
 // NEP11TokensOf returns an array of token IDs for the specified owner of the specified NFT token.
 func (c *Client) NEP11TokensOf(tokenHash util.Uint160, owner util.Uint160) ([][]byte, error) {
-	result, err := c.InvokeFunction(tokenHash, "tokensOf", []smartcontract.Parameter{
+	result, err := c.InvokeAndPackIteratorResults(tokenHash, "tokensOf", []smartcontract.Parameter{
```

Comment:
The problem here is that an account can have _a lot_ of tokens (say it's an exchange or NFT marketplace). And at the same time we have three possibilities:
 * default server that will return an unpacked (but truncated) array of items for the default `tokensOf` invocation
 * session-enabled server that can return an iterator to pick values from
 * `InvokeAndPackIteratorResults()` that can always run out of GAS (notice that it's not a problem for the session-oriented solution)

The default `NEP11TokensOf()` should probably be session-enabled, returning an iterator, because that allows to retrieve all of the elements from the appropriately configured server. At the same time we can have `NEP11UnpackedTokensOf` that will use `InvokeAndPackIteratorResults()` internally with some additional limit on the items number (client-controlled unpacking). Server-side unpacking will be used then for public nodes mostly, it's still compatible with `NEP11UnpackedTokensOf`, but other clients can invoke `tokensOf` directly and at least get something in reply.

Reply:
>  And at the same time we have three possibilities

Actually, we have four, and that's a problem. The fourth one is about the default server that will return no items for the default `tokensOf` invocation (default C# RPC). We have to support this case in our CLI (that uses `NEP11TokensOf` under the hood) that's why I made `InvokeAndPackIteratorResults`.

> The default NEP11TokensOf() should probably be session-enabled

Agree, let's create `NEP11TokensOf()` and `NEP11UnpackedTokensOf` on the client-side. But I still suggest to use `NEP11UnpackedTokensOf` for our CLI.